app/controller/command/commands/export.py

```diff
@@ -0,0 +1,194 @@
+"""Command parsing for user events."""
+import logging
+import shlex
+
+from argparse import ArgumentParser, _SubParsersAction
+from app.controller import ResponseTuple
+from app.controller.command.commands.base import Command
+from db.facade import DBFacade
+from app.model import User, Team, Permissions
+from interface.slack import Bot
+
+
+class ExportCommand(Command):
+    """Represent Export Command Parser."""
+
+    command_name = "export"
+    permission_error = "You do not have the sufficient " \
+                       "permission level for this command!"
+    lookup_error = "Lookup error!"
+    no_emails_missing_msg = "All members have emails! Nice!"
+    char_limit_exceed_msg = "WARNING! Could not export all emails for " \
+                            "exceeding slack character limits :("
+    no_user_msg = "No members found for exporting emails!"
+    no_team_found_msg = "No teams exist with the provided name!"
+    multiple_team_same_name_msg = "There are more than one team with the " \
+                                  "provided name!\n" \
+                                  "Please change the team names to be unique"
```

Comment:
This should be almost impossible to get to. Make sure that it is really hammered home. Something like ALL CAPS should do the trick nicely.